README.md

```diff
@@ -0,0 +1,136 @@
+HighLine
+========
+
+by James Edward Gray II
```

Comment:
Would it be just a link to https://github.com/JEG2/highline/graphs/contributors ?
Or do you want it verbatim? 
Who would enter the list?

I think you should preserve your name verbatim as the author! (It's different of contributor).
It was your idea with Gregory Brown? Or Gregory joined after?

Well... Summing up.... could you write exactly how you want it to be? I'm fine with anyway you like :+1: